integ/src/eks_provider.rs

```diff
@@ -124,9 +131,79 @@ pub async fn get_cluster_info(cluster_name: &str, region: &str) -> ProviderResul
         controlplane_sg,
         clustershared_sg,
         iam_instance_profile_arn,
+        dns_ip,
     })
 }
 
+async fn dns_ip(eks_client: &aws_sdk_eks::Client, cluster_name: &str) -> ProviderResult<String> {
+    let describe_results = eks_client
+        .describe_cluster()
+        .name(cluster_name)
+        .send()
+        .await
+        .context("Unable to get eks describe cluster")?;
+
+    let ip_family = describe_results
+        .cluster
+        .as_ref()
+        .context("Response missing cluster field")?
+        .kubernetes_network_config
```

Comment:
This `kubernetes_network_config` was fetched 3 times in `dns_ip()`,  does it make sense to store the reference as a variable?